node-fs-readfile/cat.js

```diff
@@ -0,0 +1,17 @@
+import { readFile } from 'node:fs/promises';
+
+const files = process.argv.slice(2);
+
+async function cat(files) {
+  try {
+    const filepaths = files.map(
+      async (item) => await readFile('./' + item, { encoding: 'utf8' })
```

Comment:
async/await here works, but it's not necessary. `readFile` returns a Promise, which is what you want to pass to Promise.all, so you can just use `(item) => readFile(...)`.